docs/BUILDING.md

```diff
@@ -33,9 +30,9 @@ _Note: Instructions assume using powershell_
 cd Shipwright
 
 # Setup cmake project
-& 'C:\Program Files\CMake\bin\cmake' -S . -B "build/x64" -G "Visual Studio 17 2022" -T v142 -A x64 # -DCMAKE_BUILD_TYPE:STRING=Release (if you're packaging)
+& 'C:\Program Files\CMake\bin\cmake' -S . -B "build/x64" -G "Visual Studio 17 2022" -T v143 -A x64 # -DCMAKE_BUILD_TYPE:STRING=Release (if you're packaging)
 # or for VS2019
-& 'C:\Program Files\CMake\bin\cmake' -S . -B "build/x64" -G "Visual Studio 16 2019" -T v142 -A x64
+& 'C:\Program Files\CMake\bin\cmake' -S . -B "build/x64" -G "Visual Studio 16 2019" -T v143 -A x64
```

Comment:
Is it even possible to install v143 with VS 2019? If not we could just drop this entirely

Reply:
I wouldn't be opposed to dropping it, though.

Reply:
With the VS 2019 build tools installer, v143 does not show up as a possible download. Older ones like v141 show up.
This support question appears to confirm that experience.
https://learn.microsoft.com/en-us/answers/questions/769911/visual-studio-2019-build-tools-v143

I would suggest dropping it entirely.

Reply:
Done.